src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -442,6 +442,21 @@ def create_base_dir(self, mode):
         except cephfs.Error as e:
             raise VolumeException(-e.args[0], e.args[1])
 
+    def _get_clone_source_info(self):
+        try:
+            src_vol_name = self.metadata_mgr.get_option('source', 'volume')
+            src_subvol_name = self.metadata_mgr.get_option('source', 'subvolume')
+            src_snap_name = self.metadata_mgr.get_option('source', 'snapshot')
+
+            return {'volume': src_vol_name, 'subvolume': src_subvol_name,
+                    'snapshot': src_snap_name}
+        except MetadataMgrException as e:
+            if e.errno == errno.ENOENT:
```

Comment:
In this case, would it help to show clone source information as N/A? Also, this seems to be invoked for non-clone subvolumes, and in that this would need to be invoked for only cloned subvolumes.

Reply:
Done.